src/messages/message.controller.ts

```diff
@@ -24,4 +24,10 @@ export class MessagesController {
     const message: Message = await this.messagesService.createMessage(createMessageDto, user.id);
     return this.messageSerializer.serialize(message);
   }
+
+  @Get("classroom/:classroomId")
```

Comment:
This should be `classrooms/:classroomId/messages`. The endpoint for the Post request should also be the same